### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by cwjolly**
p tags are used to generate the description when other methods fail.  However, there are cases where the p tag may have internal  script tags and the javascript is interpreted as text  and so is used for  the  description; this is an obvious error. This fix removes any p tag  with internal script tags from consideration when generating a description. Follett internal reference D-21669.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -5,24 +5,24 @@ var MetaInspector = require('../index'),
 
 require('./fixtures/fixtures');
 
-describe('metainspector', function(){
```
Comment (diminutivesloop) [inline test/test.js]: Let's keep the changes to the paren spacing out of this PR especially since it doesn't match the rest of the codebase.

File: package.json [hunk 2]
```diff
@@ -1,9 +1,12 @@
 {
   "name": "fss-node-metainspector",
   "description": "Npm package for web scraping purposes. You give it an URL, and it lets you easily get its title, links, images, description, keywords, meta tags",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "homepage": "http://github.com/gabceb/node-metainspector",
   "repository": "git://github.com/gabceb/node-metainspector.git",
+  "publishConfig": {
```
Comment (diminutivesloop) [inline package.json]: Since `publishConfig` is specific to our Follett fork and I'd like to submit your fix to the original repo, let's put this change in a separate PR against **patched**.

File: index.js [hunk 3]
```diff
@@ -151,6 +151,19 @@ MetaInspector.prototype.getMetaDescription = function()
 	return this;
 }
 
+MetaInspector.prototype.elemContainsTag = function(elem,tag) {
+	if (elem.name && elem.name === tag ) {
```
Comment (diminutivesloop) [inline index.js]: I think we should use `elem.tagName` as `elem.name` appears to be limited in the types of elements it applies to (see note on https://developer.mozilla.org/en-US/docs/Web/API/Element/name). Also it might be clearer if the `tag` parameter were renamed to `tagName` to indicate it's a name and not a tag object.

**Review by diminutivesloop — CHANGES_REQUESTED** [review]

File: index.js (hunk 3 above)
Comment (cwjolly) [reply index.js]: while doing tests I did not see the property tagName only  name.  Name change would be good.

File: package.json (hunk 2 above)
Comment (cwjolly) [reply package.json]: Looks like this snuck in good catch

Comment (diminutivesloop) [reply package.json]: So where are we actually configuring the npm registry? I agree it shouldn't be configured in the code, but It doesn't seem to be currently configured in the Jenkins job.

File: index.js (hunk 3 above)
Comment (diminutivesloop) [reply index.js]: Per our Slack convo let's just go ahead and switch to `elem.tagName` since it doesn't break the tests.

**Review by diminutivesloop — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (diminutivesloop) [reply package.json]: Unless I'm mistaken it looks like the Jenkins job has an `.npmrc` file configured which should set nexus as the registry, so I believe we're all set here.

File: index.js [hunk 4]
```diff
@@ -151,6 +151,19 @@ MetaInspector.prototype.getMetaDescription = function()
 	return this;
 }
 
+MetaInspector.prototype.elemContainsTag = function(elem,tagName) {
+	if (elem.tagName && elem.tagName === tagName ) {
+		return true;
+	} else if ( elem.children != undefined ) {
+		for ( let i = 0; i < elem.children.length; i++) {
```
Comment (diminutivesloop) [inline index.js]: Looks like this is causing the Travis CI build to fail since `let` is only supported in Node 6 and this library targets as far back as Node 0.10. Can you replace the `let` declaration with `var`?

**Review by diminutivesloop — CHANGES_REQUESTED** [review]

**Review by diminutivesloop — APPROVED** [review]
